### ISSUE TITLE
Tap home to take you to the top of the feed.

### ISSUE THREAD

**Issue by TurnItOff-OnAgain** [liftoff-app/liftoff#93]
Tapping the home button while already on the home page should take you to the very top of the feed.

**Comment by andybug** [comment]
I agree, after scrolling for a while it's quite a journey to get to the top to refresh.

**Comment by jordan1776** [comment]
A tap would probably be to easy to accidentally press, maybe long press or double tap the home button?

**Comment by TurnItOff-OnAgain** [comment]
My thought was to make it easier to get the drop down for different feeds. I was having issues getting it to show. I seem to have figured it out though.

@andybug For refreshing the current feed, I you can just select the same feed from the selection drop down.

Now that I think about it, calling it a drop down isn't... quite right? You tap the bar at the top which has the down carrot, but the menu pops up at the bottom. While it works, and I'm not sure should be a focus to look at in development right now, it feels like the selection being at the top, and the down carrot being up there, the menu should be at the top.

**Comment by snargledorf** [comment]
> A tap would probably be to easy to accidentally press, maybe long press or double tap the home button?

I've used plenty of apps that implement scroll to the top using the icon in the nav bar and it's never been an issue (Twitter, Sync for Reddit, Facebook, to name a few).

One thing I have seen implemented to account for accidental presses is to have a momentary "restore scroll position"  [snackbar](https://m2.material.io/components/snackbars) appear to bring you back to where you were before tapping the button.

**Comment by snargledorf** [comment]
> One thing I have seen implemented to account for accidental presses is to have a momentary "restore scroll position" [snackbar](https://m2.material.io/components/snackbars) appear to bring you back to where you were before tapping the button.

I did just come across [this comment](https://github.com/liftoff-app/liftoff/blob/e1d9bffee38feca1559581833609ad151e4429be/lib/pages/home_tab.dart#L356-L359C35) on the current scroll to top implementation. The fact that the feed is being fully refreshed would probably make not possible to implement the "restore scroll position"

**Comment by RudahXimenes** [comment]
This behavior is a default setting in many apps and, to me, this is the most annoying feature that is missing in Liftoff. I really wish that "Tap home = back to top" become a thing in the future.
